utils/test/utils_test.exs

```diff
@@ -73,6 +73,24 @@ defmodule UtilsTest do
     end
   end
 
+  test "Ensure no broken / empty links in livebooks" do
+    exercises = fetch_livebooks("../exercises/")
+    reading = fetch_livebooks("../reading/")
+
+    assert any_invalid_links?(~r/\]\(.*exercises\/\w+.livemd\)/, exercises) == false
+    assert any_invalid_links?(~r/\]\(.*reading\/\w+.livemd\)/, reading) == false
+    assert any_invalid_links?(~r/\]\(\)/, exercises ++ reading) == false
+  end
+
+  defp any_invalid_links?(regex, livebooks) do
```

Comment:
`any_invalid_links?` and `any_show_editors?` seem like they could be unified into a single `files_contain?(regex, livebooks)` function.